package.json

```diff
@@ -93,7 +92,7 @@
     "uuid": "^9.0.0",
     "warp-contracts": "^1.2.13",
     "webextension-polyfill": "^0.10.0",
-    "wouter": "^2.8.0-alpha.2"
+    "wouter": "^3.3.5"
```

Comment:
Updated to the latest version, which already includes the functionality from `path-to-regepx`.